codecov.yml

```diff
@@ -15,6 +15,8 @@ codecov:
     layout: "reach, diff, flags, files"
     behavior: default
   require_ci_to_pass: true
```

Comment:
nit: we can also remove this to ensure it never blocks CI, let me know your thoughts here!

Reply:
Resolving conversation as we have agreed to keep it!